src/main/java/de/zalando/zmon/dataservice/config/OpenTracingConfigProperties.java

```diff
@@ -0,0 +1,119 @@
+package de.zalando.zmon.dataservice.config;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import org.springframework.context.annotation.Configuration;
+
+public class OpenTracingConfigProperties {
+
+    private String tracingProvider = "noop";
```

Comment:
I would make it `enum` instead. Actually you almost implemented `enum` type with `OpenTracingProvider` class